templates/bind/zones/db.template.j2

```diff
@@ -52,6 +54,14 @@ $TTL {{ zone.negative_ttl|default('3600') }}   ; 1 hour
                     TXT     "{{ txt }}"
 {%   endfor %}
 {% endif %}
+{# CAA record is being used to specify a CA for this domain.
+   Only this CA can create certificates for the domain.
+   Also see: https://www.rfc-editor.org/rfc/rfc8659.html #}
+{% if zone.caa_records is defined %}
+{%   for caa in zone.caa_records %}
+                    CAA     {{ caa }}
+{%   endfor %}
```

Comment:
The indentation is a bit unusual to me.  Are you okay with spaces before the expression?

```
  {% for caa in zone.caa_records %}
                    CAA     {{ caa }}
  {% endfor %}
```